apps/src/code-studio/pd/workshop_enrollment/enroll_form.jsx

```diff
@@ -756,7 +760,10 @@ export default class EnrollForm extends React.Component {
 
     if (this.props.workshop_course === CSF) {
       requiredFields.push('role', 'grades_teaching');
-      if (this.props.workshop_subject === INTRO) {
+      if (
+        this.props.workshop_subject === INTRO ||
+        this.props.workshop_subject === SubjectNames.SUBJECT_CSF_DISTRICT
```

Comment:
Nit: could make this a constant at the top to be consistent with the `INTRO` and `CSF` (although I realize there are other `SubjectNames.` that haven't been extracted)